contrib/main/plugins.yml

```diff
@@ -24,3 +24,11 @@
   description: |
     Tutor plugin to install Meilisearch and enable course content search in Studio.
     See the README on GitHub for further setup instructions.
+
+- name: wordpress
+  src: -e tutor-contrib-wordpress
```

Comment:
That's just not going to work. This is going to run `pip install -e tutor-contrib-wordpress`, which fails with:

    ERROR: tutor-contrib-wordpress is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with bzr+http, bzr+https, bzr+ssh, bzr+sftp, bzr+ftp, bzr+lp, bzr+file, git+http, git+https, git+ssh, git+git, git+file, hg+file, hg+http, hg+https, hg+ssh, hg+static-http, svn+ssh, svn+http, svn+https, svn+svn, svn+file).

Reply:
Ah, I feel stupid! Fixed.

Reply:
I need some sleep! 😁
It's fixed now!